src/patternfly/sass-utilities/scss-variables.scss

```diff
@@ -144,6 +144,11 @@ $pf-global--breakpoint--md: 768px !default;
 $pf-global--breakpoint--lg: 992px !default;
 $pf-global--breakpoint--xl: 1200px !default;
 
+// Device breakpoints
```

Comment:
Do we need to add these new breakpoints. Can't we just use what we already have and have a min/max breakpoint for the mobile vs tablet and then use the xl breakpoint?